packages/backend/.eslintrc.js

```diff
@@ -20,6 +20,5 @@ module.exports = {
     '@typescript-eslint/interface-name-prefix': 'off',
     '@typescript-eslint/explicit-function-return-type': 'off',
```

Comment:
could you please quickly just explain the motivations behind these settings

Reply:
also `    '@typescript-eslint/explicit-module-boundary-types': 'off',` it got cut off

Reply:
These settings were already there, we decided to leave them apart from the rule that allowed the type any. These rules just allow interfaces to be named without an 'I' and allow functions to exist without an explicit return type.

Reply:
oh can we make functions have an explicit return please

Reply:
I think it aids a lot in readability

Reply:
or at least just `@typescript-eslint/explicit-module-boundary-types`

Reply:
`@typescript-eslint/explicit-function-return-type` is probably fine off

Reply:
'@typescript-eslint/explicit-module-boundary-types': 'off' does the same as explicit-function-return-type except for exported functions, would you like that to be enforced as well?

Reply:
yes I would prefer `@typescript-eslint/explicit-module-boundary-types` to be set to `warn`. not so fussed about `@typescript-eslint/explicit-function-return-type`